### PR TITLE
config: add unit tests for enable-multi-network

### PR DESCRIPTION
**Pull request by dcbw**

### CODE REVIEW

**Review by maiqueb — APPROVED** [review]
Thank you!!!

File: go-controller/pkg/config/config_test.go [hunk 1]
```diff
@@ -749,6 +753,7 @@ var _ = Describe("Config Operations", func() {
 			"-metrics-enable-config-duration=true",
 			"-egressip-reachability-total-timeout=5",
 			"-egressip-node-healthcheck-port=4321",
+			"-enable-multi-network=true",
```
Comment (maiqueb) [inline go-controller/pkg/config/config_test.go:756]: I've pulled your PR, and changed this line to feature only `"-enable-multi-network"` and the test still passes. 

While the test is valuable, this didn't address my particular issue. 

Thanks for the help never the less, I'll keep digging.

Comment (tssurya) [reply go-controller/pkg/config/config_test.go:756]: I am running into the same issue, I am trying to add a new flag in one of my PRs and its just not taking effect into the Config Params which is confusing..

**Review by tssurya — APPROVED** [review]
/lgtm
thanks for adding the unit tests around this flag, helped me discover we had such tests in the first place, which is nice cause now I can do the same for the new flag I am adding.

**Pull request merged**